spec/index.bs

```diff
@@ -1562,6 +1562,15 @@ A <dfn dfn>supported limits</dfn> object has a value for every limit defined by
         allowed in {{GPUPipelineLayoutDescriptor/bindGroupLayouts}}
         when creating a {{GPUPipelineLayout}}.
 
+    <tr><td><dfn>maxBindGroupsPlusVertexBuffers</dfn>
```

Comment:
Another thought I wanted to record for posterity, even though I'm pretty sure it's a terrible idea: we could, in theory, enshrine the double-ended binding implementation (bind groups grow from one end, vertex buffers grow from the other end), and *instead* of allowing unsetting, have the two calls implicitly *overwrite* one another when the limit is exceeded. This would mean the maxBindGroupsPlusVertexBuffers you set on the device directly influences whether this overwriting occurs in a given case, which is super weird. But I _think_ it's also the case that raising the limit would never break existing code, so it's not completely horrible. This would solve my concern about statefulness of the API (library calls having to potentially assume they need to unbind slots to make sure they won't error), but might introduce even weirder problems.

Anyway, just a thought.